workflows/ncn/hooks/before-each/cos-prechecks-for-worker-reboots.yaml

```diff
@@ -38,8 +38,8 @@ spec:
     ssh ${SSH_OPT} ${TARGET_NCN} "\
       if [[ -f /opt/cray/shasta/uss/bin/prechecks_for_worker_reboots ]]; then \
           /opt/cray/shasta/uss/bin/prechecks_for_worker_reboots; \
-      elif [[ -f /opt/cray/shasta/cne/bin/prechecks_for_worker_reboots ]]; then \
-          /opt/cray/shasta/cne/bin/prechecks_for_worker_reboots; \
+      elif [[ -f /opt/cray/shasta/uss/bin/prechecks_for_worker_reboots ]]; then \
+          /opt/cray/shasta/uss/bin/prechecks_for_worker_reboots; \
```

Comment:
It looks like this script is checking for the existence of a file named `prechecks_for_worker_reboots` in three different placed while trying to account for the various naming iterations of the COS/CNE/USS product, and then it's executing the one that's found. The lines above (39-40) already check for the existence of the script with `uss` in the path, so this one should be left to fall back on the `cne` location, if it exists. So I'd revert the changes to this file.

Reply:
Reverting changes in next commit